src/systemd/sd-journal.h

```diff
@@ -104,9 +104,9 @@ int sd_journal_get_seqnum(sd_journal *j, uint64_t *ret_seqnum, sd_id128_t *ret_s
 int sd_journal_set_data_threshold(sd_journal *j, size_t sz);
 int sd_journal_get_data_threshold(sd_journal *j, size_t *sz);
 
-int sd_journal_get_data(sd_journal *j, const char *field, const void **data, size_t *l);
-int sd_journal_enumerate_data(sd_journal *j, const void **data, size_t *l);
-int sd_journal_enumerate_available_data(sd_journal *j, const void **data, size_t *l);
+int sd_journal_get_data(sd_journal *j, const char *field, const void **data, size_t *size);
+int sd_journal_enumerate_data(sd_journal *j, const void **data, size_t *size);
+int sd_journal_enumerate_available_data(sd_journal *j, const void **data, size_t *size);
```

Comment:
This is problematic. The variable was `l` for "length". It shouldn't be called "size".